qemu/tests/cfg/passthrough_with_multipath_device.cfg

```diff
@@ -0,0 +1,22 @@
+- passthrough_with_multipath_device:
+    virt_test_type = qemu
+    type = passthrough_with_multipath_device
+    kill_vm_on_error = yes
```

Comment:
Please replace it with `kill_vm = yes` since `kill_vm_on_error` is dropped from framework.